test/parallel/test-assert-fail.js

```diff
@@ -5,6 +5,12 @@
 const common = require('../common');
 const assert = require('assert');
 
+common.expectWarning(
```

Comment:
Can you put the tests with the valid signatures to another file?

Reply:
Done